rpc/kate-rpc/src/lib.rs

```diff
@@ -131,6 +137,24 @@ where
 	}
 }
 
+fn app_id_from_opaque(opaque: &OpaqueExtrinsic) -> Result<AppId, String> {
```

Comment:
This fn is unused, I just left it (temporarily) to demonstrate how to only extract the AppId, without decoding the call and triggering BoundedVec limit check in native code.

Reply:
maybe to add some todo so we can delete it later if it is not used?

Reply:
Let's delete it as we do not use it.